e2e/test-util.ts

```diff
@@ -209,3 +228,15 @@ export async function deleteSession(page: Page, sessionName: string) {
   await searchSessionInfo.fill('');
   await expect(page.getByText(sessionName)).toBeHidden();
 }
+
+export async function readConfigFileFromPath(page: Page, rawPath) {
+  const filePath = path.resolve(__dirname, rawPath);
+  const mockData = await fs.readFile(filePath, 'utf-8');
+  await page.route('http://127.0.0.1:9081/config.toml', async (route) => {
```

Comment:
Please use `webuiEndpoint` from `test-util.ts` instead of `127.0.0.1:9081` in this file and `login.test.ts`.